packages/contracts/contracts/fractal_registry/FractalRegistry.sol

```diff
@@ -14,20 +14,27 @@ contract FractalRegistry {
         root = _root;
     }
 
-    /// @param addr is ERC20 address
+    /// @param addr is Eth address
     /// @return FractalId as bytes32
     function getFractalId(address addr) public view returns (bytes32) {
         return fractalIdForAddress[addr];
     }
 
-    /// @notice Adds a user to the mapping of ERC20 address to fractalId.
-    /// @param addr is ERC20 address.
+    /// @notice Adds a user to the mapping of Eth address to FractalId.
+    /// @param addr is Eth address.
     /// @param fractalId is FractalId in bytes32.
     function addUserAddress(address addr, bytes32 fractalId) public {
         requireMutatePermission();
         fractalIdForAddress[addr] = fractalId;
     }
 
+    /// @notice Removes an address from the mapping of Eth address to FractalId.
+    /// @param addr is Eth address.
+    function removeUserAddress(address addr) public {
+        requireMutatePermission();
+        delete fractalIdForAddress[addr];
+    }
```

Comment:
now that I notice, both this function and `addUserAddress` can be external instead of public, right?